jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/ScrollSpyNav.java

```diff
@@ -8,6 +9,7 @@
     @UI(".//h4 | .//h5") public ListGroup header;//@FindBy(xpath = ".//h4 | .//h5")
     @UI("p") public ListGroup mainText;          // @FindBy(css = "p")
 
+    @JDIAction("Scroll {0} to '{index}' check 'className' class exists")
```

Comment:
done

Reply:
Such way:  **{className}** or **'{className}'**

You can verify it in Console output. 
For current code:
```
...
@JDIAction("Scroll {0} to '{index}' check {'className'} class exists")
    public void scrollParagraph(ListGroup listGroup, int index, String className){
...
```
JDI will print to Console something like :
![image](https://user-images.githubusercontent.com/12463846/66936099-ce88eb00-f045-11e9-9230-87788e938a13.png)

You can see that **{'className'}** wasn't replaced by parameter **className** because it does not match pattern **{parameterName}**